src/vs/workbench/contrib/chat/browser/tools/textSearchTool.ts

```diff
@@ -14,12 +14,13 @@ import { CountTokensCallback, IPreparedToolInvocation, IToolData, IToolImpl, ITo
 import { ChatModel } from '../../common/chatModel.js';
 import { IChatService } from '../../common/chatService.js';
 
+const DEFAULT_MAX_RESULTS = 50;
```

Comment:
Thoughts on this number? The previous default was inherited from the `search.maxResults` setting which is a whopping `20000` results by default!